.circleci/config.yml

```diff
@@ -0,0 +1,54 @@
+version: 2
+jobs:
+  tests:
+    docker:
+      # Primary docker container where the tests will be run
+      - image: ubuntu:18.04
+
+      # Secondary docker container for database service
+      - image: postgres:10.6
+        environment:
+          POSTGRES_USER: circleci
+
+    working_directory: ~/hetmech-backend
+
+    steps:
+      - checkout
+      - run:
+          name: OS update
+          command: |
+            apt update && apt upgrade --yes
+      - run:
+          name: Set up Postgres
+          command: |
+            apt install postgresql-client --yes
+            createdb --host=localhost --username=circleci circleci_test
+            cp .circleci/ci_secrets.yml dj_hetmech/secrets.yml
+      - run:
+          name: Install miniconda
+          working_directory: ~/
+          command: |
+            apt install git wget --yes
+            wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh --output-document miniconda.sh
+            bash miniconda.sh -b -p $HOME/miniconda
+      - run:
+          name: Backend tests
+          command: |
+            source $HOME/miniconda/etc/profile.d/conda.sh
+            # gcc is required for "pip install git+https://..."
+            apt install gcc --yes
+            conda env create --quiet --file ./environment.yml
+            conda activate hetmech-backend
```

Comment:
Line 41 probably has to be in the current step, because the `python manage.py ..." commands depends on the conda enviironment. And the env in previous step may not be inherited by the step afterwards. Let me try it.

Reply:
Does any of these help us

+ https://circleci.com/docs/2.0/env-vars/#using-bash_env-to-set-environment-variables
+ https://stackoverflow.com/questions/50374812/circleci-2-0-does-not-add-the-environment-variable